lib/guard/plugin_util.rb

```diff
@@ -55,7 +55,7 @@ def initialize(name)
     def initialize_plugin(options)
       klass = plugin_class
       fail "Could not load class: #{_constant_name.inspect}" unless klass
-      if klass.superclass.to_s == "Guard::Guard"
```

Comment:
Because `Module` doesn't have `superclass` so it breaks. `klass` needs to be an instance of `Class` or (as I've done) have an empty `def self.superclass`.